src/index.html

```diff
@@ -11,8 +14,62 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        class="card__image"
+        alt="Picture of iMac 27 inch Retina"
+      />
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__review">
+        <div class="stars">
```

Comment:
Add the modifier class and use styles for this block from the previous task
```suggestion
        <div class="stars stars--4">
```

Reply:
Hi! Do I need to add a modifier class to each star? I'm currently using the styles from the previous task—should I change anything there, or did I make an error in that task?